cabal.project

```diff
@@ -65,6 +65,7 @@ packages:
   libs/cardano-ledger-test
   libs/plutus-preprocessor
   libs/ledger-state
+  libs/constrained
```

Comment:
We need to rename the library, since there is already a package with that name on Hackage: https://hackage.haskell.org/package/constrained

Maybe a suffix indicating that it integrates with QuickCheck?

* eg. `QuickCheck-constrained`
* or `constrained-generator`?

Reply:
I like `constrained-generators`!